tests/robot/COMPOSITION_TESTS/CREATE_COMPOSITION/B.6.a_main_flow_create_new_event_COMPOSITION.robot

```diff
@@ -40,9 +40,9 @@ Main flow create new event COMPOSITION CANONICAL_XML
     check the successful result of commit composition
 
 Main flow create new event COMPOSITION FLAT
-    [Tags]    future
+    [Tags]    not-ready
```

Comment:
please provide an JIRA or Github issue ID, so that we know later why this test is marked as not-ready

Reply:
changes done